test/FunctionalTests/Microsoft.OData.Core.Tests/NavigationPropertyOnComplexTests.cs

```diff
@@ -1674,5 +1675,28 @@ public void NullValidationErrorMessageForCollectionsofComplexTypes()
             }
             Assert.Equal(expectedErrorMessage, actualErrorMessage);
         }
+
+        [Fact]
+        public void NullValidationErrorMessageForANullableCollectionsofComplexTypes()
+        {
+            string expectedErrorMessage = "A null value was found for the property named 'AddressesNullable', which has the expected type 'Collection(DefaultNs.Address)[Nullable=True]'. The expected type 'Collection(DefaultNs.Address)[Nullable=True]' does not allow null values.";

```

Comment:
```suggestion
            string expectedErrorMessage = "A null value was found for the property named 'AddressesNullable', which has the expected type 'Collection(DefaultNs.Address)[Nullable=True]'. For an empty collection of type 'Collection(DefaultNs.Address)[Nullable=True]', set the value to '[]' instead of null.";

```
Could you change this message? It doesn't make itclear why a field that is marked nullable cannot not be set to null.

Reply:
Noted.